### PR TITLE
Testing and LGTM un-regressions

### PR DESCRIPTION
**Pull request by mattwelborn**
## Description
This PR:
1) Removes the xfail from tests involving the QCA server now that it is updated to >=0.12.0. 
2) Runs `make format`
3) Removes `LGTM py/unused-import` annotations since Semmle/ql#2014 is resolved.

## Changelog description
This PR does not need a change log line.

## Status
- [x] Ready to go

### CODE REVIEW

File: qcfractal/interface/collections/torsiondrive_dataset.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 from .collection_utils import register_collection
 
 if TYPE_CHECKING:  # pragma: no cover
-    from ..models import Molecule  # lgtm[py/unused-import] (https://github.com/Semmle/ql/issues/2014)
```
Comment (mattwelborn) [inline qcfractal/interface/collections/torsiondrive_dataset.py]: It looks like autoflake is unaware of `if TYPE_CHECKING` in the same way that LGTM was. I guess we're going to have to substitute the lgtm annotation for a pyflakes annotation...

Comment (dgasmith) [reply qcfractal/interface/collections/torsiondrive_dataset.py]: Maybe turn off `auto flake` for now? Its nice, but not needed.

File: qcfractal/interface/collections/dataset.py [hunk 2]
```diff
@@ -10,10 +10,9 @@
 import pandas as pd
 import requests
 from pydantic import Field, validator
-from tqdm import tqdm
-
 from qcelemental import constants
 from qcelemental.models.types import Array
+from tqdm import tqdm
```
Comment (dgasmith) [inline qcfractal/interface/collections/dataset.py]: Unsure why this order was changed?

Comment (mattwelborn) [reply qcfractal/interface/collections/dataset.py]: I think it alphabetizes.

**Pull request merged**